_episodes/03-create.md

```diff
@@ -198,15 +198,14 @@ draft.txt
 > ## Creating Files a Different Way
 >
 > We have seen how to create text files using the `nano` editor.
-> Now, try the following command in your home directory:
+> Now, try the following command:
 >
 > ~~~
-> $ cd                  # go to your home directory
 > $ touch my_file.txt
 > ~~~
 > {: .language-bash}
 >
-> 1.  What did the touch command do?
+> 1.  What did the `touch` command do?
 >     When you look at your home directory using the GUI file explorer,
```

Comment:
This should refer to the *current* directory, not the *home* directory.

Reply:
Thanks, how could I miss that?!